ethexe/ethereum/src/abi/events/router.rs

```diff
@@ -36,15 +36,6 @@ impl From<IRouter::CodeGotValidated> for RouterEvent {
     }
 }
 
-impl From<IRouter::CodeValidationRequested> for RouterEvent {
```

Comment:
I can't implement `From<(Router::Code Validation Requested, Bytes32)> for RouterEvent` due to orphan rule

Reply:
:(